src/components/modules/auth/ui/forms/LoginForm.tsx

```
@@ -1,6 +1,8 @@
 import GoogleSignInButton from "@/components/modules/auth/ui/google/GoogleSignInButton";
 import { Button } from "@/components/ui/button";
 import { Input } from "@/components/ui/input";
+import GoogleSignInButton from "@/components/modules/auth/ui/google/GoogleSignInButton";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'GoogleSignInButton'. Consider to delete it or rename it.

'GoogleSignInButton' is defined here:


(lint/suspicious/noRedeclare)